cpp_macros/src/lib.rs

```diff
@@ -279,10 +309,15 @@ NOTE: They cannot be generated by macro expansion."#,
         }
     };
 
-    let rust_invocations = parsing::find_all_rust_macro(synom::ParseState::new(&closure.body.node))
-        .expect("rust! macro");
+    use std::iter::FromIterator;
+    use syn::parse::Parser;
+    let input = proc_macro2::TokenStream::from_iter(vec![closure.body].into_iter());
```

Comment:
nit: It's unfortunate you have to do `from_iter` here, but you should be able to just use a fixed size array rather than `vec!`